lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -334,13 +334,16 @@ class GradebookView(GradeViewMixin, PaginatedAPIView):
         GET /api/grades/v1/gradebook/{course_id}/                       - Get gradebook entries for all users in course
         GET /api/grades/v1/gradebook/{course_id}/?username={username}   - Get grades for specific user in course
         GET /api/grades/v1/gradebook/{course_id}/?user_contains={user_contains}
+        GET /api/grades/v1/gradebook/{course_id}/?user_contains={username_contains}
         GET /api/grades/v1/gradebook/{course_id}/?cohort_id={cohort_id}
         GET /api/grades/v1/gradebook/{course_id}/?enrollment_mode={enrollment_mode}
     **GET Parameters**
         A GET request may include the following query parameters.
         * username:  (optional) A string representation of a user's username.
         * user_contains: (optional) A substring against which a case-insensitive substring filter will be performed
           on the USER_MODEL.username, or the USER_MODEL.email, or the PROGRAM_ENROLLMENT.external_user_key fields.
+        * username_contains: (optional) A substring against which a case-insensitive substring filter will be performed
```

Comment:
This is backwards. The username_contains search only searches usernames. user_contains searches all of those fields.